### PR TITLE
Completed Issue 2 view entries

### PR DESCRIPTION
**Pull request by maryalenn**
code-journal-issue-2

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -1,2 +1,96 @@
...
+//     <p>$$inputNotes</p>
+//    </div>
+//  </div>
+// </li> */
+
+// Define the parent element
+var $entriesList = document.getElementsByClassName('entries-list');
+// Create the first child element and append it to parent element
+var $listItem = document.createElement('li');
+$entriesList[0].appendChild($listItem);
```
Comment (BrettAlbright) [inline js/main.js]: you should be rendering `li`s in the render function, you dont want to do something like this out in the non function space. you can eliminate this code

File: js/main.js [hunk 2]
```diff
@@ -1,2 +1,96 @@
...
+
+// Define the rendering function
+function renderEntries(entries) {
+
```
Comment (BrettAlbright) [inline js/main.js:49]: li should be created here, you need 1 for each entry

File: js/main.js [hunk 3]
```diff
@@ -1,2 +1,96 @@
...
+  $entryNotes.innerHTML = entries.notes;
+
+  return $listItem;
+}
+
+for (var i = 0; i < data.entries.length; i++) {
+  // eslint-disable-next-line no-unused-vars
+  var $entry = renderEntries(data.entries[i]);
+  // eslint-disable-next-line no-unused-expressions
+  $entriesList.appendChild;
```
Comment (BrettAlbright) [inline js/main.js]: this is not doing anything

File: js/main.js [hunk 4]
```diff
@@ -1,2 +1,96 @@
...
+
+// function to insert the value of the input URL into the placeholder image src)
+function updateImage(event) {
+  // update the class of the placeholder image to .entry-image. Remove placeholder class & alt text
+  $entryImage.classList.add('entry-image');
+  $entryImage.alt = '';
+  $entryImage.src = event.target.value;
+  $inputImageURL = event.target.value;
+  // update the src attribute of the image to the $inputImageURL value
+  return $entryImage;
```
Comment (BrettAlbright) [inline js/main.js]: dont need to return anything here

**Review by Zhiwutian — CHANGES_REQUESTED** [review]

File: js/main.js (hunk 1 above)
Comment (maryalenn) [reply js/main.js]: Updated render function so that the `li` is created within the function (lines 51, 54 of main.js)

File: js/main.js (hunk 3 above)
Comment (maryalenn) [reply js/main.js]: removed

File: js/main.js (hunk 4 above)
Comment (maryalenn) [reply js/main.js]: removed return statement

**Pull request merged**